packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -149,6 +245,49 @@ const PreviousDateRangePickerImpl = ({
 		}
 	}
 
+	const handleTimeChange = (
+		event: React.ChangeEvent<HTMLInputElement>,
+		input: 'start' | 'end',
+	) => {
+		const value = event.target.value
+		const isValid = value.match(VALID_FORMATTED_TIME_REGEX) !== null
```

Comment:
can also use moment for this too? https://stackoverflow.com/a/40131561 - then we can get rid of the regex constant

Reply:
thanks!